docs/design-proposals/multi-tenancy/multi-tenancy-network.md

```diff
@@ -238,6 +241,17 @@ Network object specifies service IPAM as *external*:
       service:
         ipam: external
 
+#### User hints
+
+User can provide hints about service IP address.
+
+For Arktos managed pool, it will look at the specific annotation of service object like below
+```arktos.futurewei.com/vip-hint: {"cidr":"10.10.1.0/26"}```
+
+Other keys supported are "ip", "policy" (with "BestEffort" the default, or "Guaranteed").
```

Comment:
How is ip gets passed? ```arktos.futurewei.com/vip-hint: {"cidr":"10.10.1.0/26", "ip": "10.10.1.8", "policy": "BestEffort"}``` Like this?

Reply:
reasonable guess :) yes. If ip is specified, no need to have cidr; but this does not matter too much for the topic.